freespace2/SDLGraphicsOperations.cpp

```diff
@@ -69,6 +69,14 @@ std::unique_ptr<os::OpenGLContext> SDLGraphicsOperations::createOpenGLContext(co
 	SDL_GL_SetAttribute(SDL_GL_MULTISAMPLEBUFFERS, (attrs.multi_samples == 0) ? 0 : 1);
 	SDL_GL_SetAttribute(SDL_GL_MULTISAMPLESAMPLES, attrs.multi_samples);
 
+	const int gl_versions[] = { 41, 40, 33, 32, 31, 30, 21, 20 };
```

Comment:
Do we actually require an OpenGL 4.1 context? IMO it would be better to stick to one OpenGL version (e.g. 3.3 or 3.2) and if additional functionality is required we obtain that by using the corresponding extension if it exists.